Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/Bits.h

```diff
@@ -239,9 +240,9 @@ namespace AZ
         /**
         * Returns the value divided by alignment, where the result is rounded up if the remainder is non-zero.
         */
-        template <typename T> inline T DivideByMultiple(T value, size_t alignment)
+        template <typename T> inline T DivideByMultiple2(T value, size_t alignment)
```

Comment:
question: I take it this is the call soon to be removed? Name could instead potentially be `DivideByMultiple_DEPRECATED` or `DivideByMultipleDeprecated` to be more explicit? Not a huge deal though and this isn't well defined/documented either so leaving as-is isn't a big deal

Reply:
It seems there's a chance this could break since DivideByMultiple2 is templated and DivideAndRoundUp is not.

Reply:
Should the overloads handle this though? Thought maybe not every case for all sizes of `int`